### ISSUE TITLE
Generalization of Overparametrized Deep Neural Network Under Noisy Observations 

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#156]
## 一言でいうと
鈴木先生が書かれてそうな論文だ
このあたりの話ちゃんと理解したい @JumpeiNagase 一緒に読みませんか？

### 論文リンク
https://openreview.net/forum?id=bZJbzaj_IlP

## 概要
本研究では、Rectified Linear Unit (ReLU)アクティベーションを用いたオーバーパラメータ化ディープニューラルネットワーク(DNN)の一般化特性を研究する。
ノンパラメトリック回帰の枠組みの下で、ReLU DNNのニューラルタンジェントカーネル(NTK)によって誘導される再生カーネルヒルベルト空間(RKHS)からのグランドトゥルース関数を仮定し、データセットはノイズで与えられる。早期停止を微妙に採用することなく、バニラ勾配降下法によって学習されたオーバーパラメトリックDNNは、グランドトゥルース関数を回復しないことを証明する。これにより、推定されたDNNの予測誤差は.から離れて拘束されることが判明した。

上記の結果を補完するものとして、-regularized gradient descentによって、オーバーパラメトリックDNNが早期に停止することなく、予測誤差のミニマックス最適収束率を達成できることを示す。注目すべきは、我々が得た収束率は、文献で知られているよりも速いということである。
